### PR TITLE
De-duplicate the dartanalyzer command output

### PR DESCRIPTION
**Pull request by devoncarew**
- partially address https://github.com/flutter/flutter/issues/9438
- de-duplicate the dartanalyzer command output (a more complete solution is here: https://codereview.chromium.org/2793813002/ ; waiting to land until after 1.23)

@pq @danrubel

### CODE REVIEW

**Review by pq — APPROVED** [review]
LGTM

**Comment by Hixie** [comment]
LGTM

File: dev/bots/test.dart [hunk 1]
```diff
@@ -118,7 +118,7 @@ Future<Null> _pubRunTest(
   String workingDirectory, {
   String testPath,
 }) {
-  final List<String> args = <String>['run', 'test'];
+  final List<String> args = <String>['run', 'test', '-rexpanded'];
```
Comment (Hixie) [inline dev/bots/test.dart]: was this intentional?

Comment (devoncarew) [inline dev/bots/test.dart]: Yes - to help see what tests were running on the bots. Here's the before: https://travis-ci.org/flutter/flutter/jobs/223314813#L5740, and the after: https://travis-ci.org/flutter/flutter/jobs/223317672#L5804.

**Pull request merged**